create_teacher.rb

```diff
@@ -11,5 +12,6 @@ def create_teacher
 
   teacher = Teacher.new(age, specialization, name)
   @people.push(teacher)
+  save_teacher(age, specialization, name)
```

Comment:
- [ ] you are saving to the json file each time the teacher gets created, the issue with this is that after adding the teacher and then showing the list of teachers the added teacher will appear twice. you can solve this by updating all the json files when the user is closing the app.